docs/guides/docker.md

```diff
@@ -8,224 +8,216 @@ author:
-#
-# Firstly, remove `depends_on:` and `network_mode:` from the Transmission and SABnzbd containers. After that you'll need to move the `ports:`
-# section from the Wireguard container and add it to the Transmission and SABnzbd containers. The unused "default Wireguard listening port"
-# line can also be safely removed.
 #
-# 2. Docker Compose's `volume parameter`
+# 1. Docker Compose's `volume parameter`
 #
 # The `volume` parameter lets you mount directories from your host system to the Docker container so that information can persist upon bringing a container
-# down or up.
+# down or up. Create these directories where you want and add them for each container following the template.
+# See (https://docs.docker.com/engine/storage/) for more information on giving your containers access to storage locations.
+# Generally, the structure follows a `/path/to/your/data:/path/your/container/sees` structure, where the path after the colon is how the folders you add will appear from the perspective of your container.
 #
-# Following the recommend directory structure from before, we've already filled out the volume mount settings, so that every service will have it's
-# own directory for it's compose files, all in the form of `/home/user/projects/automation/<service name>`.
+# Following the recommend directory structure from before, we've already filled out the volume mount settings, so that every service will have its
+# own directory for its compose files, all in the form of `/home/user/projects/automation/<service name>`.
 #
 # This can of course be changed if you're experienced with Docker and would prefer to use custom mount locations.
 #
-# 3. Values for PUID and PGID
+# 2. Values for PUID and PGID
 #
 # These values refer to the host OS user's UID and GID, respectively. You can find this out by entering `id $user` in your terminal.
 #
-# 4. Timezones
+# 3. Timezones
 #
 # All instances of `TZ` in the `environment` section should be changed to `TZ=Your/Timezone`. To find your timezone and the correct way to format it
 # for Docker, you can check this Wikipedia article (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) and get the information
 # from the "TZ identifier" column
+#
+# 4. Gluetun and VPN providers
+#
+# environment variables and setup instructions for each provider can be found at (https://github.com/qdm12/gluetun-wiki/tree/main/setup/providers)
+# To help with readability, settings have been filled out to roughly match a configuration for ProtonVPN.
 
 ---
 version: "3.8"
 services:
-  wireguard:
-    container_name: wireguard
-    image: linuxserver/wireguard
-    environment:
-      - PUID=1000
-      - PGID=1000
-      - TZ=Europe/London
+  gluetun:
+    container_name: gluetun
+    image: qmcgaw/gluetun:latest # This tag points to the development version due to dev preferences. If having stability issues, relpace :latest with :v3
```

Comment:
I can see this being a problem in the future when v3 is no longer relevant. I understand the nuance on the :latest vs :v3 subject but it might be better long-term to either recommend :latest regardless, as we're already doing that for every other container present. I don't see the potential drawbacks of using :latest as justification to add extra complications to end users reading this guide who will probably not be experienced with Docker, containers, registries, changing images, etc.